pkg/messaging/v1/util.go

```diff
@@ -51,7 +50,7 @@ const (
 
 // DaprInternalMetadata is the metadata type to transfer HTTP header and gRPC metadata
 // from user app to Dapr.
-type DaprInternalMetadata map[string]*structpb.ListValue
+type DaprInternalMetadata map[string]*internalv1pb.ListStringValue
```

Comment:
This doesn't have to be a pointer, right?

Reply:
Yes it must be a pointer. protoc generates `map[string]*ListStringValue` in go for `map<string, ListStringValue>`. please check pb.go files.